manage/config-dialog.js

```diff
@@ -197,6 +197,10 @@ function configDialog(style) {
     renderValues();
   }
 
+  function isNumber(value) {
+    return value !== null && value !== '' && !isNaN(Number(value));
```

Comment:
This would fail with:
```js
const foo = {
  toString: () => ""
};
isNumber(foo); // true
```

Reply:
And how would you provide a toString function in usercss meta header?

Reply:
You can't, but it doesn't mean the function is doing the right thing.

Reply:
Do you mean it can fail on the real input? Or do you mean it's theoretically wrong? Because I don't really care about the latter.

Reply:
> Do you mean it can fail on the real input?

I'm not sure what is "real input". Currently, this function is only used to check `va.min`/`va.max`/`va.step`.

> do you mean it's theoretically wrong? Because I don't really care about the latter.

Then let's change it to `return typeof value === 'number'`. It is shorter and will work with `va.min`/`va.max`/`va.step`. Also, it would match the function name so you don't have to rename it to `isNumeric`.

Reply:
I think what @eight04 is trying to say is that `va.min`, `va.max` and `va.step` will always be numeric values since only the units label is the only allowed string value while setting the variable. All other strings are filtered out; therefore, we don't need to use `Number()`.

This means we can change the function to:

```js
function isNumber(value) {
  return typeof value === 'number';
}
```

I've been parsing the values over-cautiously. So I'll switch it to the above function.

Reply:
There's no need for a function then...

Reply:
Ok, all number checks are inline now.